languages/go/sample-basics.go

```diff
@@ -0,0 +1,84 @@
+
+package main
+
+import "fmt"
+
+func main() {
+  fmt.Println("+++ Sample Basics in Go +++")
+
+  println("Generating sample slice of ints...")
+  sample_numbers := []int{1, 2, 3, 4, 5, 10, 100}
+  println(stringify(sample_numbers))
+
+  println("Generating sample slice of strings...")
+  sample_strings := []string{"abc", "123", "xyz"}
+  println(stringify(sample_strings))
+
+  println("Summing sample ints...")
+  sample_numbers_sum := sum(sample_numbers...)
+  println(stringify(sample_numbers_sum))
+
+  println("Concatenating sample strings...")
+  sample_strings_concat := concat(sample_strings...)
+  println(stringify(sample_strings_concat))
+
+  println("Building and running sample tests...")
```

Comment:
Note: If this were real-life production code, we wouldn't be building/executing our own tests like this. Instead we'd be using proper assert-style libraries and frameworks. For this sample though, I was just exploring simple roll-your-own approaches for demonstration purposes.